### DISCUSSION TITLE
hide three dot for preview

### DISCUSSION THREAD

**Discussion by anditsung** [laravel/nova-issues#3917] [Ideas & Feature Requests]
if the user dont have the permission to view the resource

<img width="1047" alt="Screen Shot 2022-04-09 at 11 32 36" src="https://user-images.githubusercontent.com/1090413/162556449-35d504a9-2a22-4213-83a9-01b048a91827.png">
<img width="1042" alt="Screen Shot 2022-04-09 at 11 33 42" src="https://user-images.githubusercontent.com/1090413/162556456-f9820553-7c4f-468e-804e-d2cd43bd3288.png">

**Comment by wdelfuego** [comment]
I reported this as a bug, should be fixed in the next release 

https://github.com/laravel/nova-issues/issues/3925

**Reply by anditsung** [reply]
Detail page have this three dot icon too

**Reply by wdelfuego** [reply]
Let the developers know in the issue I linked, I only reported the bug

**Reply by wdelfuego** [reply]
Or better yet, wait for the release that contains the fix and create a new bug report if the issue still exists on the Detail view
